packages/cli/src/PublicApi/v2/openapi.yml

```diff
@@ -0,0 +1,231 @@
+---
+openapi: 3.0.0
+info:
+  title: Public n8n API
+  description: n8n Public API
+  termsOfService: https://n8n.io/legal/terms
+  contact:
+    email: hello@n8n.io
+  license:
+    name: Apache 2.0 with Commons Clause
+    url: https://github.com/n8n-io/n8n/blob/master/packages/cli/LICENSE.md
+  version: 1.0.0
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+servers:
+  - url: /api/v2
+tags:
+- name: users
+  description: Operations about user
+  externalDocs:
+    description: Find out more about our store
+    url: http://swagger.io
+paths:
+  /users:
+    get:
+      x-eov-operation-id: getUsers
+      x-eov-operation-handler: v1/handlers/Users
+      tags:
+      - users
+      summary: Retrieve all users
+      description: Retrieve all users from your instance. Only available for the instance owner.
+      parameters:
+      - name: limit
+        in: query
+        description: The maximum number of items to return
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: number
+          example: 100
+          default: 100
+      - name: cursor
+        in: query
+        description: Paginate through users by setting the cursor parameter to a nextCursor attribute returned by a previous request's response_metadata. Default value fetches the first "page" of the collection. See pagination for more detail.
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: string
+          example: MTIzZTQ1NjctZTg5Yi0xMmQzLWE0NTYtNDI2NjE0MTc0MDA
+      - name: includeRole
+        in: query
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: boolean
+          example: true
+      responses:
+        "200":
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/inline_response_200'
```

Comment:
Don't see the benefit of using $ref here. Why not really inlining the schema (instead of referencing it but calling the schema inline_response_200, which also sounds very generic but is in fact scoped to the user endpoint)?!